src/test/java/com/exasol/sql/expression/function/exasol/ExasolCastFunctionTest.java

```diff
@@ -0,0 +1,27 @@
+package com.exasol.sql.expression.function.exasol;
+
+import static com.exasol.hamcrest.SqlFragmentRenderResultMatcher.rendersTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import org.junit.jupiter.api.Test;
+
+import com.exasol.datatype.type.Varchar;
+import com.exasol.sql.StatementFactory;
+import com.exasol.sql.dql.select.Select;
+import com.exasol.sql.expression.NullLiteral;
+
+class ExasolCastFunctionTest {
+
+    @Test
+    void testRendering() {
+        final Select select = StatementFactory.getInstance().select().cast(NullLiteral.nullLiteral(), new Varchar(254));
```

Comment:
I just thought that it's not really a good idea to have a select in the function test. I know that other tests also have it, but maybe it's better to test only function rendering without select. And a select test should go to the select class

Reply:
yes.